### PR TITLE
remove MUI circularProgress & grid

### PR DESCRIPTION
**Pull request by ayushjnv1**
## Proposed Changes

- Fixes #issue?
- Change 1
- Change 2
- More?

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by gigincg** [comment]
@ayushjnv1 Thank you for the PR

Could you please, merge the upstream to resolve the conflicts & update the PR description and link to the Issues you are solving

**Comment by ayushjnv1** [comment]
https://github.com/coronasafe/care_fe/issues/4930 this is the issue that I fixed

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@ayushjnv1 @rithviknishad @gigincg 

image

- [ ] the forgot password text should be left aligned similar to the existing staging
- [ ] the login button should be of the same length as the text box, similar to the existing staging
- [ ] by default, English is selected, so it should be much more highlighted from others and have an underline to make the user understand, the language English is selected, similar to the existing staging.

File: src/Components/Auth/Login.tsx [hunk 1]
```diff
@@ -391,7 +391,7 @@ export const Login = (props: { forgot?: boolean }) => {
                   size="large"
                   className="font-extrabold"
                 />
-                <Grid container justify="center">
+                <div className="grid justify-center">
```
Comment (rithviknishad) [inline src/Components/Auth/Login.tsx]:
```suggestion
                <div className="justify-start">
```

File: src/Components/Auth/Login.tsx [hunk 2]
```diff
@@ -307,15 +307,15 @@ export const Login = (props: { forgot?: boolean }) => {
                   size="large"
                   className="font-extrabold"
                 />
-                <Grid container justify="center">
+                <div className="grid justify-center ">
```
Comment (rithviknishad) [inline src/Components/Auth/Login.tsx]:
```suggestion
                <div className="justify-start">
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]
1. The grid classes are unnecessary.
2. Make button `w-full`

**Review by rithviknishad — APPROVED** [review]
@ayushjnv1 LGTM.

@nihal467 the language selector issue is a different issue not related to this.

What happens is, if the user has never selected a language in care, it'll be blank and English is not highlighted. Because only when a user selects a language, the preference is stored in the local storage.
In your staging, English is highlighted because you've selected English at some point in time. 

This issue is easily noticeable in all deploy previews.

In my fresh staging (see image) I've never selected a language, so it's not highlighted. Will create an issue for this.

image

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**